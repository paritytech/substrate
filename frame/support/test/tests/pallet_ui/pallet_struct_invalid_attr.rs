#[frame_support::pallet]
mod pallet {
	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	#[pallet::generate_storage_info] // invalid
	pub struct Pallet<T>(_);

	#[pallet::call]
	impl<T: Config> Pallet<T> {}
}

fn main() {
}
