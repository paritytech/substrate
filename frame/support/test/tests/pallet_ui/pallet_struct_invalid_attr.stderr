error: expected one of: `generate_store`, `without_storage_info`, `storage_version`
 --> tests/pallet_ui/pallet_struct_invalid_attr.rs:7:12
  |
7 |     #[pallet::generate_storage_info] // invalid
  |               ^^^^^^^^^^^^^^^^^^^^^
